tests/lib/rules/local-dependency.js

```diff
@@ -0,0 +1,337 @@
+/**
+ * @fileoverview Checks existence of local dependencies in the package.json
+ * @author Kendall Gassner
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+const path = require('path');
+var rule = require('../../../lib/rules/local-dependency'),
+    RuleTester = require('eslint').RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const fileName = partialPath => {
+    return path.join(process.cwd(), partialPath);
+};
+
+var ruleTester = new RuleTester();
+ruleTester.run('local-dependency', rule, {
+    valid: [
+        {
+            code: `module.exports = {
+                    "name": "pandages",
+                    "version": "1.0.0",
+                    "description": "",
+                    "main": "index.js",
+                    "keywords": [],
+                    "author": "me!",
+                    "license": "ISC",
```

Comment:
[Cleanup] The other test files include a lot of unnecessary fields, but especially for this one with a lot of tests (which is awesome btw, love the test coverage!) I'm finding it hard to read through them. Would you be up for removing the unnecessary-for-tests fields?

```suggestion
```